generators/client/templates/angular/_package.json

```diff
@@ -157,7 +157,7 @@
     "e2e": "protractor <%= TEST_SRC_DIR %>protractor.conf.js",
     "postinstall": "webdriver-manager update && node node_modules/phantomjs-prebuilt/install.js && <%= clientPackageManager %> run webpack:build"
     <%_ } else { _%>
-    "postinstall": "node node_modules/phantomjs-prebuilt/install.js && <%= clientPackageManager %> run webpack:build"
+    "postinstall": "node node_modules/phantomjs-prebuilt/install.js"
```

Comment:
Yes but if someone tries to launch app without running yarn start then it would not work